ComponentKit/Core/CKComponent.mm

```diff
@@ -164,12 +167,15 @@ - (void)unmount
 
 - (void)_relinquishMountedView
 {
-  UIView *view = _mountInfo->view;
-  if (view) {
-    CKAssert(view.ck_component == self, @"");
-    [_scopeHandle.controller component:self willRelinquishView:view];
-    view.ck_component = nil;
-    _mountInfo->view = nil;
+  CKAssertMainThread();
+  if (_mountInfo != nullptr) {
```

Comment:
Somehow internally in hosting views we end up with what appears to be a re-entrant relinquish call. It won't hurt to add this call, since its equivalent actually used to exist in this code path.